docs/faq.md

```diff
@@ -0,0 +1,252 @@
+
+When using SageMaker backend or tuning remotely, we use [SageMaker checkpoint mechanism](https://docs.aws.amazon.com/sagemaker/latest/dg/model-checkpoints.html) under the hood to sync local checkpoints to s3. Checkpoints are synced to  `s3://{sagemaker-default-bucket}/syne-tune/{tuner-name}/{trial-id}/`, where `sagemaker-default-bucket` is the default bucket for SageMaker (can also be configured).
+
+### <a name="schedulers-checkpointing"></a> Which schedulers make use of checkpointing?
+
+Checkpointing means storing the state of a trial (i.e., model parameters, optimizer or learning rate scheduler parameters), so that it can be paused and potentially resumed at a later point in time, without having to start training from scratch. Syne Tune checkpointing support is detailed in https://github.com/awslabs/syne-tune/blob/main/docs/benchmarks.md#checkpointing and https://github.com/awslabs/syne-tune/blob/main/examples/training_scripts/checkpoint_example/checkpoint_example.py. Once checkpointing to and from a local file is added to a training script, Syne Tune is managing the checkpoints (e.g., copy to/from cloud storage for distributed back-ends).
+
+The following schedulers make use of checkpointing:
+
+* Promotion-based Hyperband: `HyperbandScheduler(type='promotion', ...)`
+    The code runs without checkpointing, but in this case, any trial which is resumed is started from scratch. For example, if a trial was paused after 9 epochs of training and is resumed later, training starts from scratch and the first 9 epochs are wasted effort. Moreover, extra variance is introduced by starting from scratch, since weights may be initialized differently. It is not recommended to run promotion-based Hyperband without checkpointing.
+* Population-based training: `PopulationBasedTraining`
+    PBT does not work without checkpointing.
+* Synchronous Hyperband: `SynchronousGeometricHyperbandScheduler`
+    This code runs without checkpointing, but wastes effort in the same sense as promotion-based asynchronous Hyperband
+
+### <a name="tuner-checkpointing"></a> Is the tuner checkpointed?
+
+Yes. When performing the tuning, the tuner state is regularly saved on the experiment path under `tuner.dill`
+(every 10 seconds which can be configured with `results_update_interval`).
+This allows to use spot-instances when running a tunning remotely with the remote launcher. It also allows to 
+resume a past experiment or analyse the state of scheduler at any point.
+
+### <a name="trial-output"></a> Where can I find the output of my trials?
+
+When running  `LocalBackend` locally, results of trials are saved under `~/syne-tune/{tuner-name}/{trial-id}/` and contains the following files:
```

Comment:
You should add that if the local backend is run remotely, all this information will be synced to S3, even including the logs and checkpoints.

Reply:
This is a nice answer here. To me, it feels that it subsumes a few of the others. You here say what is stored for the experiment, and in which subdirectories. This includes results, model checkpoints (optional), and logs, the latter per trial. For remote tuning and local backend, all of this is synced to S3 periodically.